frontend/src/components/key-value-table/key-value-table.tsx

```diff
@@ -0,0 +1,32 @@
+import type { HTMLAttributes, ReactNode } from 'react'
+
+import { Colors } from '../../config/colors'
+
+interface KeyValueTableProps extends HTMLAttributes<HTMLTableElement> {
+  rows: Array<{ key: ReactNode; value: ReactNode; dataTestId?: string }>
+}
+
+export const KeyValueTable = ({ rows, style }: KeyValueTableProps) => {
+  return (
+    <table style={{ fontSize: 14, ...style }}>
```

Comment:
For key-value pairs <dl> is more semantically correct
ref. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl